core/templates/pages/admin-page/activities-tab/admin-prod-mode-activities-tab.component.ts

```diff
@@ -13,23 +14,24 @@
 // limitations under the License.
 
 /**
- * @fileoverview Directive for the activities tab in the admin panel when Oppia
+ * @fileoverview Component for the activities tab in the admin panel when Oppia
  * is in production mode.
  */
 
-require('domain/utilities/url-interpolation.service.ts');
+import { Component } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+
+
+@Component({
+  selector: 'admin-prod-mode-activities-tab',
+  templateUrl: './admin-prod-mode-activities-tab.component.html',
+  styleUrls: []
+})
+
+export class AdminProdModeActivitiesTabComponent {
```

Comment:
@srijanreddy98, Do we use the Suffix `Component` in the class name? (not sure!)

Note:

Earlier the directive names didn't have any suffix also the decorator on the top explains that the following class definition is a component. (I'm not sure what we are going to follow but just wanted to add a few thoughts.)

Reply:
Yes, class names should have a suffix depicting what they are for. When we are in the file we can see the decorator, but when this class is imported in the module (or any other component, inca ase of component lazy loading), the class name should depict what is the class for.
This is in mentioned in the angular style guide (https://angular.io/guide/styleguide#symbols-and-file-names)

Reply:
Ah I see, I was confused with the naming in angularJS (where we didn't had this prefix). Thanks for the explanation)